files/postfix/userdb-aliases.cf

```diff
@@ -0,0 +1,6 @@
+hosts = postgresql://userdb-attempt-too-do-user-989073-0.db.ondigitalocean.com:25060/userdb?sslmode=require
+user = mail
+password = userdb-mail-lookup
```

Comment:
Does this need magic?

Reply:
it needs:
- a user that can read name and host from passwd (already created)
- the `postfix-pgsql` package installed (which should be handled in [docker-postfix#14](https://github.com/hashbang/docker-postfix/pull/14))